tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionDetailRepositoryTests.cs

```diff
@@ -51,11 +51,6 @@ await CatalogueItemEntityBuilder
 
             await SolutionEntityBuilder.Create()
                 .WithId(Solution1Id)
-                .Build()
-                .InsertAsync();
-
-            await SolutionDetailEntityBuilder.Create()
-                .WithSolutionId(Solution1Id)
                 .WithAboutUrl("AboutUrl")
                 .WithFeatures("Features")
                 .Build()
```

Comment:
I can't highlight the correct line, due to GitHub limitations, but rather than call `InsertAndSetCurrentForSolutionAsync` we should call `InsertAsync`.

`InsertAndSetCurrentForSolutionAsync` can then be removed from `SolutionEntity`.

The same applies elsewhere (line 116, for example).

Reply:
Replaced `InsertAndSetCurrentForSolutionAsync` with `InsertAsync`